client/core/core.go

```diff
@@ -2565,12 +2586,34 @@ func (c *Core) Trade(pw []byte, form *TradeForm) (*Order, error) {
 	return corder, nil
 }
 
+// validateFeeRateLimit checks if user provided fee rate limit value, if yes
+// it ensures that the server'r MaxFeeRate is smaller than the wallet's
+// configured feeRateLimit.
+func validateFeeRateLimit(w *xcWallet, a dex.Asset) error {
```

Comment:
For clarity in the calling code, it's best to pass what's needed and not more.  That is, 
`func validateFeeRateLimit(feeRateLimit, maxFeeRate uint64) error {` But the function is so trivial I'd be inclined to just do the logic in the caller.  The only thing that warrants the function is the `feeRateLimit != 0` check and it seems that's not possible since the default is non-zero, right?

Reply:
true, that check was there before adding the default will get red of it